### DISCUSSION TITLE
【已解决】Alist搭建完部署CF CDN之后 webdav提示403

### DISCUSSION THREAD

**Discussion by Yorkcn** [AlistGo/alist#3288] [Q&A]
Alist搭建完部署CF CDN之后 webdav客户端链接提示403，这个是什么原因啊，麻烦大佬告知

**Comment by yugu91** [comment]
请问您怎么解决？我套了CloudFont也是提示403，但是我套CloudFlare正常

**Reply by Yorkcn** [reply]
我是因为宝塔防火墙误报问题，将/dav路径添加白名单就好了
